src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_rpc_plugin.cc

```diff
@@ -47,8 +47,8 @@ bool RCRPCPlugin::Init(
     policy::PolicyHandlerInterface& policy_handler) {
   const auto& settings = app_manager.get_settings();
   const auto frequency = settings.get_vehicle_data_frequency();
-  interior_data_cache_.reset(new InteriorDataCacheImpl(frequency.second));
-
+  interior_data_cache_.reset(new InteriorDataCacheImpl());
+  interior_data_cache_->StartRequestResetTimer(frequency.second);
```

Comment:
@ZhdanovP why can't we start timer in InteriorDataCacheImpl constructor?

Reply:
@LuxoftAKutsan it's better to move start in separate method for two reasons: 
1. It's allow to change time frame without killing the interior cache object
2. It's allow to test this method in unit tests